### PR TITLE
Update remote-machine-learning.md

### PR DESCRIPTION
**Pull request by kirilldem**
provide an example to use cuda or another container. Copied from official documentation

### CODE REVIEW

File: docs/docs/guides/remote-machine-learning.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ version: '3.8'
 services:
   immich-machine-learning:
     container_name: immich_machine_learning
+    # For hardware acceleration, add one of -[armnn, cuda, openvino] to the image tag.
+    # Example tag: ${IMMICH_VERSION:-release}-cuda
     image: ghcr.io/immich-app/immich-machine-learning:${IMMICH_VERSION:-release}
     volumes:
```
Comment (aviv926) [inline docs/docs/guides/remote-machine-learning.md:22]:
```suggestion
   extends:
      file: hwaccel.ml.yml
      service: # set to one of [armnn, cuda, openvino, openvino-wsl] for accelerated inference - use the `-wsl` version for WSL2 where applicable
    volumes:
```

Comment (kirilldem) [reply docs/docs/guides/remote-machine-learning.md:22]: I am not an expert at these but for it to work i did not need to add this line :) someone else can probably give their opinion

Comment (mertalev) [reply docs/docs/guides/remote-machine-learning.md:22]: It will run without that line, but won't actually use CUDA.

Comment (mertalev) [reply docs/docs/guides/remote-machine-learning.md:22]: Is the indentation in that suggestion right? It looks a bit off.

Comment (aviv926) [reply docs/docs/guides/remote-machine-learning.md:22]: > Is the indentation in that suggestion right? It looks a bit off.

I copied it from my computer and it looks fine, maybe it just looks like that on GitHub.

Comment (mertalev) [reply docs/docs/guides/remote-machine-learning.md:22]: I think you need to add another space before `extends`

File: docs/docs/guides/remote-machine-learning.md [hunk 2]
```diff
@@ -16,7 +16,12 @@ version: '3.8'
 services:
   immich-machine-learning:
     container_name: immich_machine_learning
+    # For hardware acceleration, add one of -[armnn, cuda, openvino] to the image tag.
+    # Example tag: ${IMMICH_VERSION:-release}-cuda
     image: ghcr.io/immich-app/immich-machine-learning:${IMMICH_VERSION:-release}
+    extends:
+      file: hwaccel.ml.yml
+      service: # set to one of [armnn, cuda, openvino, openvino-wsl] for accelerated inference - use the `-wsl` version for WSL2 where applicable
```
Comment (mertalev) [inline docs/docs/guides/remote-machine-learning.md:24]:
```suggestion
    # extends:
    #   file: hwaccel.ml.yml
    #   service: # set to one of [armnn, cuda, openvino, openvino-wsl] for accelerated inference - use the `-wsl` version for WSL2 where applicable
```

**Review by mertalev — APPROVED** [review]

**Pull request merged**